CHANGELOG.md

```diff
@@ -111,6 +111,22 @@
 4. Fixed issue where creating FQDN gateway with ``fqdn_lan_interface`` causes replacement
 
 
+## 2.23.3 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.8.1455**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Site2Cloud
+1. Implemented support for remote identification using empty string in the following resources:
+    - **aviatrix_site2cloud**
+    - **aviatrix_spoke_external_device_conn**
+    - **aviatrix_transit_external_device_conn**
+
+### Bug Fixes:
+1. Fixed issue where FQDN tag source_ip_list requires executing terraform apply twice for more than 2 x GWs in **aviatrix_fqdn**
```

Comment:
Fixed issue where FQDN tag's`source_ip_list` requires executing terraform apply twice for more than 2 gateways